client/apps/eternum-mobile/src/features/ws-chat/lib/filterUtils.ts

```
@@ -43,5 +43,5 @@ export const filterRoomsBySearch = (rooms: Room[], searchText: string): Room[] =
 export const filterUsersBySearch = (users: User[], searchText: string): User[] => {
   if (!searchText.trim()) return users;
 
-  return users.filter((user) => (user.username || user.id).toLowerCase().includes(searchText.toLowerCase()));
+  return users?.filter((user) => (user?.username || user?.id).toLowerCase().includes(searchText.toLowerCase()));
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)